unlock-app/app/og/event/[slug]/route.tsx

```diff
@@ -0,0 +1,197 @@
+                    <rect
+                      width="32"
+                      height="32"
+                      fill="white"
+                      transform="translate(16 16)"
+                    />
+                  </clipPath>
+                </defs>
+              </svg>
+              <p tw="text-2xl ml-6t font-semibold w-96">{startTime}</p>
+            </div>
+          )}
+
+          {location && (
+            <div tw="flex items-center mt-6 bg-white/50 rounded-2xl">
+              <svg
+                width="64"
+                height="64"
+                viewBox="0 0 64 64"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <rect
+                  x="0.5"
+                  y="0.5"
+                  width="63"
+                  height="63"
+                  rx="15.5"
+                  fill="#FFFDFA"
+                />
+                <g clip-path="url(#clip0_403_6549)">
+                  <path
+                    d="M32 43.8669L38.6 37.2669C39.9052 35.9616 40.794 34.2985 41.1541 32.4881C41.5141 30.6776 41.3292 28.801 40.6228 27.0956C39.9163 25.3902 38.7201 23.9326 37.1852 22.9071C35.6504 21.8816 33.8459 21.3342 32 21.3342C30.1541 21.3342 28.3496 21.8816 26.8148 22.9071C25.2799 23.9326 24.0837 25.3902 23.3772 27.0956C22.6708 28.801 22.4859 30.6776 22.8459 32.4881C23.206 34.2985 24.0948 35.9616 25.4 37.2669L32 43.8669ZM32 47.6376L23.5147 39.1522C21.8365 37.474 20.6936 35.3358 20.2306 33.008C19.7676 30.6803 20.0052 28.2675 20.9135 26.0748C21.8217 23.8821 23.3598 22.0079 25.3332 20.6893C27.3066 19.3708 29.6266 18.667 32 18.667C34.3734 18.667 36.6934 19.3708 38.6668 20.6893C40.6402 22.0079 42.1783 23.8821 43.0865 26.0748C43.9948 28.2675 44.2324 30.6803 43.7694 33.008C43.3064 35.3358 42.1636 37.474 40.4853 39.1522L32 47.6376V47.6376ZM30.6667 29.3336V25.3336H33.3333V29.3336H37.3333V32.0002H33.3333V36.0002H30.6667V32.0002H26.6667V29.3336H30.6667Z"
+                    fill="#373A3E"
+                  />
+                </g>
+                <rect
+                  x="0.5"
+                  y="0.5"
+                  width="63"
+                  height="63"
+                  rx="15.5"
+                  stroke="#E4E4E4"
+                />
+                <defs>
+                  <clipPath id="clip0_403_6549">
+                    <rect
+                      width="32"
+                      height="32"
+                      fill="white"
+                      transform="translate(16 16)"
+                    />
+                  </clipPath>
+                </defs>
+              </svg>
+              <p tw="text-2xl ml-6t font-semibold w-96">{location}</p>
+            </div>
+          )}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+export async function GET(
+  request: Request,
+  { params }: { params: { slug: string } }
+) {
+  // Loading fonts from static assets
+  const inter400 = await fetch(
```

Comment:
Couldn't find a way to get fonts "locally" so we're downloading them from the public folder... which is kind of weird.